AdminWebsite/AdminWebsite/ClientApp/src/app/work-allocation/manage-team/manage-team.component.ts

```diff
@@ -0,0 +1,12 @@
+import { Component, OnInit } from '@angular/core';
+
+@Component({
+    selector: 'app-manage-team',
+    templateUrl: './manage-team.component.html',
+    styleUrls: ['./manage-team.component.scss']
```

Comment:
as this css file is empty, we could remove `styleUrls` and delete the stylesheet to fix the code smell

Reply:
yeah can do. I left it there expecting this to be picked up next but this can be brought back in when the dev work starts